index.html

```diff
@@ -0,0 +1,1380 @@
+                </div>
+                <h6 class="card-title"><a class="font-weight-bold text-decoration-none" href="#">Where Can Americans
+                    Travel Now? A Guide to Countries Open to U.S. Tourists</a></h6>
+              </div>
+
+              <div class="card w-100">
+                <div class="card-custom-img max-height-custom w-100">
+                  <img class="img-fluid w-100" src="./media/image/news(2).png" alt="news week image">
+                </div>
+                <h6 class="card-title"><a class="font-weight-bold text-decoration-none" href="#">Where Can Americans
+                    Travel Now? A Guide to Countries Open to U.S. Tourists</a></h6>
+              </div>
+
+              <div class="card w-100">
+                <div class="card-custom-img max-height-custom w-100">
+                  <img class="img-fluid w-100" src="./media/image/news(2).jpg" alt="news week image">
+                </div>
+                <h6 class="card-title"><a class="font-weight-bold text-decoration-none" href="#">Where Can Americans
+                    Travel Now? A Guide to Countries Open to U.S. Tourists</a></h6>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="news-signup d-block col-10 col-sm-7 col-lg-12 mx-auto">
+          <form action="#" class="news-signup-form jumbotron jumbotron-fluid py-3 px-4 d-flex flex-column">
+            <div class="form-header d-flex">
+              <div class="form-header-icon mr-3"></div>
+              <h6 class="form-header-title font-weight-bolder">START YOUR DAY WITH OUR TOP 5 ARTICLES</h6>
+            </div>
+
+            <input type="email" name="email" id="news-email" placeholder="Email Address">
+            <button class="btn main-bg text-white mt-3 p-1 text-uppercase font-weight-bolder" type="submit">free sign up
+              &#10095;</button>
+          </form>
+        </div>
+      </div>
+    </div>
+
+    <div class="news-magazine mt-md-5 pl-3">
+      <div class="news-magazine-title d-flex justify-content-center w-100 border border-top-0 border-left-0 border-right-0 pb-2">
+        <div class="form-header-icon p-0"></div>
+        <h5 class="text-body font-weight-bolder ml-2">IN THE MAGAZINE</h5>
+      </div>
+      <div class="row d-flex align-items-start w-100 pt-0">
+        <div class="card col-12 col-md-3 first-element pl-2 pl-sm-1">
+          <div class="card-custom-img w-sm-100">
+            <img class="img-fluid w-100" src="./media/image/mag1.jpg" alt="news week image">
+            <h1 class="card-img-overlay main-color">WORLD</h1>
+            <div class="card-footer w-100">
+              <h6 class="text-center font-weight-bolder"> SEE ALL FEATURES &#10095;</h6>
+            </div>
+          </div>
+        </div>
+
+        <div class="card col-12 col-md-3 pt-4 px-2">
+          <img class="mag-images" src="./media/image/mag(1).jpg" alt="news week image" >
```

Comment:
- [ ] **[OPTIONAL]** So that your images in this section doesn't look stretched out, you can use the property: `object-fit: cover` that way they will preserve their proportion